### ISSUE TITLE
Module not in deno.land

### ISSUE THREAD

**Issue by Verzach3** [lino-levan/raylib-deno#1]
In the example the module is imported from deno.land but the package is not there
![image](https://github.com/lino-levan/raylib-deno/assets/54605781/c1a8fac6-37da-4bf2-8c07-29538ad79ec6)

**Comment by lino-levan** [comment]
Yep, I need to put it on somewhere. I think I'll most likely be putting it on the JSR but we'll see.

**Comment by lino-levan** [comment]
Finally put it up: https://jsr.io/@lino/raylib
